src/routes/+page.svelte

```diff
@@ -13,12 +14,15 @@
   <div class="cards">
     <img src={cards} alt="cards spelling BLEND" />
   </div>
-  <a class="btn" href="https://app.blendreading.com" target="_blank" rel="noopener noreferrer"><h2>start blending</h2></a>
+  <div class="side-by-side">
+    <a class="btn" href="https://app.blendreading.com" target="_blank" rel="noopener noreferrer"><h2>start blending</h2></a>
+    <a class="btn" href="/login"><h2>sign up</h2></a>
```

Comment:
I have been feeling like it's too hard to find where to sign up for a new account (even though we have it in the nav bar) for people so i thought it would be nice just to have it right there as the very first thing on the site. hopefully this results in more people taking action and signing up

Reply:
I like it, but I think having it a different color may look better? The two buttons side by side don't look great to me. Here it is with the `btn-green` class, maybe this but a bit more transparent?
![image](https://github.com/CSMA-Technology/blend-product-site/assets/10676387/21524993-0390-4a52-8c83-a53b2dd99546)